lntypes/hash.go

```diff
@@ -8,6 +8,9 @@ import (
 // HashSize of array used to store hashes.
 const HashSize = 32
 
+// ZeroHash is a predefined hash containing all zeroes.
+var ZeroHash = Hash{}
```

Comment:
👍 

Alternatively, it can be declared as just: 
```go
var ZeroHash Hash
```

There're likely many other places that can use this new value, but we can defer updating all those locations for another change.

Reply:
Declared like suggested. Indeed not updating everywhere to keep this pr contained.